cfn-resources/autogen/schema-gen/main.go

```diff
@@ -21,13 +23,21 @@ const (
 	url                = "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git"
 	MongoDBAtlasPrefix = "MongoDB::Atlas::"
 	Unique             = "Unique"
+	OpenAPISpecPath    = "https://www.mongodb.com/8c07de79-53d6-41d8-8fc8-bacdf7f271fa"
```

Comment:
Note:  I suppose that's the URL you get on the UI, which doesn't represent the actual location of the resource on the server. This may change anytime. You need to check with MongoDB to provide a real endpoint to the doc.

Reply:
@SuperMohit , This is yet to fix, right now @Zuhairahmed  provided below URL

https://www.mongodb.com/docs/atlas/reference/api-resources-spec/

Reply:
@Zuhairahmed it seems that the url https://www.mongodb.com/8c07de79-53d6-41d8-8fc8-bacdf7f271fa is not working. is there a way to Download the specs?

Reply:
https://mongodb-mms-prod-build-server.s3.amazonaws.com/openapi/bfb9c50bdcffe3100e37c3f7b73b52915701c98b.json